template/test/e2e/nightwatch.conf.js

```diff
@@ -1,45 +1,93 @@
 require('babel-register')
 var config = require('../../config')
 
+const paths = {
+  chromedriver: require('chromedriver').path,
+  geckodriver: require('geckodriver').path,
+  seleniumServer: require('selenium-server').path,
+  launchUrl: 'http://localhost:' + (process.env.PORT || config.dev.port),
+  logPath: 'logs',
+}
+console.log('paths:', paths)
```

Comment:
Is this debug code?

Reply:
The `paths` object is not debug code. I intentionally collected all the paths for readability and to simplify the configuration.

The `console.log` is debug code and is extremely useful when trying to figure out why the webdrivers aren't loading.

I just pushed a small update (ad1999d) that:
* adds a debugTests constant to `nightwatch.conf.js`
* logs the `paths` object when it's `true`
* sets the nightwatch config object's `silent` property to `!debugTests`

I also cleaned up the file a bit.